cmd/policy-assistant/pkg/matcher/traffic.go

```diff
@@ -59,26 +59,95 @@ func labelsToString(labels map[string]string) string {
 }
 
 func (t *Traffic) PrettyString() string {
+	var src string
+	var dst string
 	if t == nil || t.Source == nil || t.Destination == nil {
 		return "<undefined>"
 	}
 
-	src := t.Source.Internal.Workload
-	if src == "" {
-		if t.Source.Internal == nil {
-			return "<undefined>"
+	if t.Source.Internal == nil && t.Destination.Internal == nil {
```

Comment:
src string should be independent of dst string right? Could we refactor this to be something like this?
```go
var src string
if t.Source != nil {
    src := t.Source.Internal.PrettyString()
}
var dst string
if t.Destination != nil {
    dst := t.Destination.Internal.PrettyString()
}
return fmt.Sprintf("%s -> %s:%d (%s)", src, dst, t.ResolvedPort, t.Protocol)
 ```